rand-core/src/lib.rs

```diff
@@ -81,6 +81,14 @@ pub mod le;
 /// in this trait directly, then use the helper functions from the [`impls`]
 /// module to implement the other methods.
 /// 
+/// It is recommended that implementations also implement:
+/// 
+/// - `Debug` but such that no internal details are output
+/// - `Serialize` and `Deserialize` (from Serde), possibly behind a feature gate
```

Comment:
Ah, something we talked a lot about :smile:. Is `behind a feature gate` not mostly a practical matter, to keep the number of dependencies down?

Reply:
It seems good practice for a library to keep dependencies optional where possible. Maybe I should expand the comment.

Reply:
Or not mention it at all, seems like mixing concerns... Your choice.